### PR TITLE
Replace release name and use local helm

### PR DESCRIPTION
**Pull request by gabwow**
This change removes all refernce to 24.0X and reference to the missing helm repo.

### CODE REVIEW

File: helm/README.md [hunk 1]
```diff
@@ -93,7 +88,7 @@ kubectl create namespace inference-ms
 A command like the one below will then use the latest chart version to install the version of NIM defined in the values file into the `inference-ms` namespace in your Kubernetes cluster. Modify it as required.
 
 ```bash
-helm --namespace inference-ms install my-nim nim-demo/nim-llm -f path/to/your/custom-values.yaml
+helm --namespace inference-ms install my-nim nim-llm -f path/to/your/custom-values.yaml
```
Comment (crookedstorm) [inline helm/README.md:91]: nit: if you make it `nim-llm/` then it looks more definite that you are using a local directory. Same for helm show values (which honestly doesn't apply as much for a chart you just downloaded anyway).

**Review by crookedstorm — APPROVED** [review]
Looks good. Feel free to address comments as you like.

File: helm/README.md (hunk 1 above)
Comment (gabwow) [reply helm/README.md]: Sure, done

File: helm/README.md [hunk 2]
```diff
@@ -93,7 +88,7 @@ kubectl create namespace inference-ms
 A command like the one below will then use the latest chart version to install the version of NIM defined in the values file into the `inference-ms` namespace in your Kubernetes cluster. Modify it as required.
 
 ```bash
-helm --namespace inference-ms install my-nim nim-demo/nim-llm -f path/to/your/custom-values.yaml
+helm --namespace inference-ms install my-nim nim-llm/ -f path/to/your/custom-values.yaml
```
Comment (supertetelman) [inline helm/README.md:91]: In another PR let's create a `config` directory with an example values.yaml, .gitignore it, and assume users are using that custom-values.yaml file in all example commands.

Comment (gabwow) [reply helm/README.md:91]: I'll put that up today

**Pull request closed without merging**